spec.html

```diff
@@ -30384,6 +30404,50 @@ <h1>Runtime Semantics: GetSubstitution ( _matched_, _str_, _position_, _captures
         </emu-clause>
       </emu-clause>
 
+      <emu-clause id="sec-string.prototype.replaceall">
+        <h1>String.prototype.replaceAll ( _searchValue_, _replaceValue_ )</h1>
+        <p>When the `replaceAll` method is called with arguments _searchValue_ and _replaceValue_, the following steps are taken:</p>
+        <emu-alg>
+          1. Let _O_ be ? RequireObjectCoercible(*this* value).
+          1. If _searchValue_ is neither *undefined* nor *null*, then
+            1. Let _isRegExp_ be ? IsRegExp(_searchValue_).
+            1. If _isRegExp_ is true, then
```

Comment:
(It looks kind of like, instead of adding asterisks to the `true` in S.p.replaceAll, you removed them from the `*true*` in S.p.matchAll, which is then what @ljharb commented on ~8 hours ago. So this one still needs to be fixed.)